src/librustdoc/lib.rs

```diff
@@ -201,6 +201,18 @@ fn opts() -> Vec<RustcOptGroup> {
                         Markdown file or generated documentation",
                        "FILES")
         }),
+        unstable("raw-js-in-header", |o|  {
+            o.optopt("", "raw-js-in-header",
```

Comment:
Bikeshed warning: Since the included file eventually gets named `additional.js`/`additional.css`, i feel like these flags would be better named `--additional-js`/`--additional-css`.

I also feel like the flag description is a little misleading. Maybe something like:

> (JS|CSS) file to include as a static file to be linked from generated documentation

(I feel like this feature would be much better served if it allowed multiple files, but that can be added in a separate PR. Since the flag will be unstable, we can edit it after it lands.)